recipes/dash_mantine_components/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dash-mantine-components" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dash_mantine_components-{{ version }}.tar.gz
+  sha256: 8a9a6f038cfa130f70043dbcd88d1baab1d4f90c9a8c6725804565eed489996f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - pip
+  run:
+    - python >=3.6
+    - setuptools
+    - dash >=2.0.0
+    - dash-core-components ==2.0.0
+    - dash-html-components ==2.0.0
+    - dash-table ==5.0.0
+    - flask >=1.0.4
+    - flask-compress
+    - importlib-metadata ==4.8.3  # [py<37]
```

Comment:
Selectors can't be used with `noarch: python` recipes. As conda-forge has dropped support for Python 3.6 and it's already end-of-life I'd be inclined to suggest changing the minimum dependency to `python >=3.7`

Reply:
Thank for the feedback. I have made the corrections.